lib/Service/Mail/Mailer.hs

```diff
@@ -137,8 +137,8 @@ makeConnection True host (Just port) = SMTPSSL.doSMTPSSLWithSettings host settin
   where
     settings = SMTPSSL.defaultSettingsSMTPSSL {SMTPSSL.sslPort = fromIntegral port}
 
-sendEmail :: [String] -> TL.Text -> [MIME.Part] -> AppContextM ()
-sendEmail to subject [] = return () -- empty mail won't be sent
+sendEmail :: [String] -> TL.Text -> [MIME.Part] -> AppContextM (Either String ())
+sendEmail to subject [] = return $ Left "Tried to sent empty email" -- empty mail won't be sent
```

Comment:
Please put it to the `Localization` file

Reply:
Fixed in https://github.com/ds-wizard/dsw-server/pull/56/commits/9138cc713207b11a3590d895176d5d1558b8b19d